frameworks/Python/fastapi/app_orjson.py

```diff
@@ -0,0 +1,116 @@
+import asyncio
+import asyncpg
+import os
+import jinja2
+from fastapi import FastAPI
+from fastapi.responses import HTMLResponse, ORJSONResponse, PlainTextResponse
+from random import randint
+from operator import itemgetter
+
+
+READ_ROW_SQL = 'SELECT "randomnumber" FROM "world" WHERE id = $1'
```

Comment:
When reading from `world` we want to select the `randomnumber` and the `id`. (See first rule [here](https://github.com/TechEmpower/FrameworkBenchmarks/wiki/Project-Information-Framework-Tests-Overview#single-database-query))

If you noticed this wasn't being done somewhere else, would you mind updating that as well.